src/App.tsx

```diff
@@ -1,99 +1,99 @@
-import React from 'react';
+import React, {
+  useCallback,
+  useEffect,
+  useMemo,
+  useState,
+} from 'react';
+import { debounce } from 'lodash';
```

Comment:
Fix it
<img width="754" alt="image" src="https://github.com/mate-academy/react_autocomplete/assets/64750958/1ff7b41a-d10c-4e5c-8b7e-bebc591a21ca">